theme/js/script.js

```diff
@@ -38,6 +38,8 @@ $(document).ready(function() {
 
     addActiveClassToNavbar();
 
+    getJSONdata();
```

Comment:
Rename this function to represent what its actually doing, getJSONData is too vague.

Reply:
done.